packages/react-core/src/components/Menu/examples/Menu.md

```diff
@@ -18,109 +18,168 @@ import AngleLeftIcon from '@patternfly/react-icons/dist/esm/icons/angle-left-ico
 
 ## Examples
 
-### Basic
+### Basic menus
+
+A menu may contain multiple variations of `<MenuItem>` components. The following example shows a few different states of menu items, where they may:
+
+- Use the `itemId` property to link to callbacks. In this example, the `onSelect` property logs information to the console when a menu item is selected. In practice, specific actions can be linked to `onSelect` callbacks.
+- Use the `to` property to direct users to other resources or webpages after selecting a menu item, and the `onClick` property to pass in a callback for specific menu items.
+- Use the `isDisabled` property to disable a menu item. 
+- Use the `isPlain` property to remove the outer box shadow and style the menu plainly instead. 
 
 ```ts file="MenuBasic.tsx"
 ```
 
 ### Danger menu item
 
+To indicate that a `<MenuItem>` is connected to a potentially destructive action, use the `isDanger` property. The "Delete" item in the following example demonstrates how danger items look.
+
 ```ts file="MenuDangerMenuItem.tsx"
 ```
 
 ### With icons
 
+Use the `icon` property to add a familiar icon before a `<MenuItem>` to accelerate text label recognition.
+
 ```ts file="MenuWithIcons.tsx"
 ```
 
-### With checkbox
+### With actions
 
-```ts file="./MenuWithCheckbox.tsx"
-```
+To connect a menu item to an action icon, add a `<MenuItemAction>` to a `<MenuItem>`, and use the `icon` property to load an easily recognizable icon.  
 
-### Filtering with search input
+To trigger an action when any menu action icon is clicked, pass a callback to the `onActionClick` property of the `<Menu>`. The following example logs to the console any time any action icon is clicked. 
 
-```ts file="MenuFilteringWithSearchInput.tsx"
+To trigger an action when a specific item's action icon is clicked, pass in the `onClick` property to that `<MenuItemAction>`. The following example logs "clicked on code icon" to the console when the "code" icon is clicked. 
```

Comment:
Does this action only happen when clicked? What if we said "is selected" instead of clicked to include screen readers or people not using a mouse?